src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -89,49 +63,81 @@ def generate_requests(
                 )
         return request_params
 
+    @staticmethod
+    def assign_placeholders(value: str, param_values: Dict[str, Any]) -> str:
+        """
+        Finds all the placeholders (indicated by <>) in the passed in value
+        and replaces them with the actual param values
+        """
+        if value:
+            placeholders = re.findall("<(.+?)>", value)
+            for placeholder in placeholders:
+                value = value.replace(
+                    f"<{placeholder}>", str(param_values[placeholder])
+                )
+        return value
```

Comment:
Should there be validation when creating/updating the saas config to make sure placeholders have corresponding values in request_params?  I'd like to be as helpful as possible when the saas config is being created.

Reply:
I'll and a logger statement for now and create a separate ticket for validating the different places the placeholders can be used.